src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -846,7 +848,7 @@ private void HandleHostError(Exception exception)
         {
             if (exception == null)
             {
-                throw new ArgumentNullException(nameof(exception));
+                throw new ArgumentNullException("exception");
```

Comment:
Curious about the reason for this change.

Reply:
This was an inadvertent revert when carving up branches - will definitely fix!